helm/stacks-blockchain/values.yaml

```diff
@@ -0,0 +1,360 @@
+    # - name: FOO
+    #   value: "bar"
+
+  ## Specify resource limits and requests for the node's container
+  ##
+  resources: {}
+    # limits:
+    #   cpu: 500m
+    #   memory: 512Mi
+    # requests:
+    #   cpu: 250m
+    #   memory: 128Mi
+
+  ## Rollback limit
+  ##
+  revisionHistoryLimit: 10
+
+  # The update strategy to apply
+  ##
+  updateStrategy: {}
+  #  rollingUpdate:
+  #    maxUnavailable: 1
+  #  type: RollingUpdate
+
+  # minReadySeconds to avoid killing pods before it's ready
+  ##
+  minReadySeconds: 0
+
+  ## Node labels for node pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: []
+  #  - key: "key"
+  #    operator: "Equal|Exists"
+  #    value: "value"
+  #    effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+  ## Affinity and anti-affinity
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+  # # An example of preferred pod anti-affinity, weight is in the range 1-100
+  # podAntiAffinity:
+  #   preferredDuringSchedulingIgnoredDuringExecution:
+  #   - weight: 100
+  #     podAffinityTerm:
+  #       labelSelector:
+  #         matchExpressions:
+  #         - key: app.kubernetes.io/name
+  #           operator: In
+  #           values:
+  #           - stacks-blockchain
+  #       topologyKey: kubernetes.io/hostname
+
+  # # An example of required pod anti-affinity
+  # podAntiAffinity:
+  #   requiredDuringSchedulingIgnoredDuringExecution:
+  #   - labelSelector:
+  #       matchExpressions:
+  #       - key: app.kubernetes.io/name
+  #         operator: In
+  #         values:
+  #         - stacks-blockchain
+  #     topologyKey: "kubernetes.io/hostname"
+
+  ## Total time it takes for the Container to stop normally
+  ##
+  terminationGracePeriodSeconds: 60
+  
+  ## Additional volumes for the node
+  ##
+  volumes: []
+  # - name: stacks-node-chain
+  #   persistentVolumeClaim:
+  #     claimName: stacks-node-chain
+
+  ## Additional volumeMounts for the node
+  ##
+  volumeMounts: []
+  # - name: stacks-node-chain
+  #   mountPath: /root/stacks-node
+
+  ## Additional containers to run alongside the node. Useful if adding a sidecar
+  ##
+  extraContainers: []
+  # - name: stacks-node-sidecar
+  #   image: my-sidecar-image:latest
+  #   imagePullPolicy: IfNotPresent
+  #   resources:
+  #     requests:
+  #       memory: "16Mi"
+  #       cpu: "50m"
+  #     limits:
+  #       memory: "32Mi"
+  #       cpu: "100m"
+
+  ## Containers which are run before the node container is started
+  ##
+  initContainers: []
+  # - name: init-myservice
+  #   image: busybox
+  #   command: ['sh', '-c', 'until nslookup myservice; do echo waiting for myservice; sleep 2; done;']
+
+################################
```

Comment:
I think we should add some links to our docs in here, specifically in regard to the configfile, but possibly in other areas as well to add some more context for the values a user can set

Reply:
Added a link to our docs and a link to some example config files